ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js

```diff
@@ -55,20 +55,22 @@ export default function SkipSRPBackup({ handleClose }) {
           {t('skipAccountSecurity')}
         </Typography>
         <Box justifyContent={JUSTIFY_CONTENT.CENTER} margin={3}>
-          <Checkbox
-            className="skip-srp-backup-popover__checkbox"
-            onClick={() => {
-              setChecked(!checked);
-            }}
-            checked={checked}
-            dataTestId="skip-srp-backup-popover-checkbox"
-          />
-          <Typography
-            className="skip-srp-backup-popover__details"
-            variant={TYPOGRAPHY.h7}
-          >
-            {t('skipAccountSecurityDetails')}
-          </Typography>
+          <label className="skip-srp-backup-popover__label">
+            <Checkbox
+              className="skip-srp-backup-popover__checkbox"
+              onClick={() => {
```

Comment:
we can just remove the curly brackets and keep this a one liner right

Reply:
Updated!